dev/diffs/3.4.3.diff

```diff
@@ -7,7 +7,7 @@ index d3544881af1..26ab186c65d 100644
      <ivy.version>2.5.1</ivy.version>
      <oro.version>2.0.8</oro.version>
 +    <spark.version.short>3.4</spark.version.short>
-+    <comet.version>0.5.0-SNAPSHOT</comet.version>
```

Comment:
How was this even working? Perhaps this is unused?

Reply:
Ah, there was a corresponding incorrect version in `setup-spark-builder`:

```
  comet-version:
    description: 'The Comet version to use for Spark'
    required: true
    default: '0.5.0-SNAPSHOT'
```

Reply:
We also set the comet version dynamically:

```
../mvnw -nsu -q versions:set-property -Dproperty=comet.version  -DnewVersion=${{inputs.comet-version}} -DgenerateBackupPoms=false
```

Reply:
It is perhaps the best to separate the PR for this 0.6.0 fix otherwise, branch-0.6 is left in a wrong state

Reply:
I filed https://github.com/apache/datafusion-comet/pull/1405 to fix the issue in branch-0.6

Reply:
@kazuyukitanimura I pushed another commit to remove all of the hard-coded Comet version number from GitHub workflows/actions.